SU-SWS/stanford_basic
# D8CORE-2004 JS Error in stanford_basic local tabs.

# READY FOR REVIEW

# Summary
- Resolve JS error by checking for `role--authenticate` class.

# Review By (Date)
- asap (05/11)

# Urgency
- high

# Steps to Test
1. Pull this branch
2. Run `npm run publish`
3. Check out console log of local instance as anonymous user.

# Affected Projects or Products
- Stanford_basic

# Associated Issues and/or People
- [D8CORE-2004](https://stanfordits.atlassian.net/browse/D8CORE-2004)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
